orders/views.py

```diff
@@ -57,4 +57,25 @@ def delete(self, request, order_item):
 
         order_item.delete()
 
-        return JsonResponse({'message':'SUCCESS'}, status=204)
\ No newline at end of file
+        return JsonResponse({'message':'SUCCESS'}, status=204)
+
+    @login_decorator
+    def get(self, request):
+        order, created = Order.objects.get_or_create(
+            user            = request.user,
+            order_status_id = OrderStatus.BASKET
+        )
+
+        order_items = [{
+                'order_item_id': order_item.id,
+                'name'         : order_item.product.name,
+                'count'        : order_item.count,
+                'price'        : order_item.product.price,
+                'stock'        : order_item.product.stock,
+                'image_url'    : order_item.product.imageurl_set.order_by('id')[0],
+                'selected'     : order_item.selected
+            }for order_item in order.orderitem_set.all()]
```

Comment:
```suggestion
            }for order_item in OrderItem.objects.filter(조건)]
```
결국 저희는 OrderItem 중에 장바구니 조건을 만족하는 제품들을 보여주는 것이므로 위와 같이 수정하면 좋을 것 같아요

Reply:
수정했습니다!